embedded/web/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@element-hq/element-call-embedded",
+  "version": "0.0.0",
+  "files": [
+    "README.md",
+    "LICENSE-AGPL-3.0",
+    "LICENSE-COMMERCIAL",
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/element-hq/element-call.git"
```

```suggestion
    "url": "git+https://github.com/element-hq/element-call.git"
```

You can run `npm pkg fix` to fix it, otherwise it'll get fixed on every release for the release artifact only